pkg/btc-wallet/src/index.js

```diff
@@ -8,7 +8,7 @@ import './css/indigo-static.css';
 import './css/fonts.css';
 import './css/custom.css';
 
-// rebuild x2
+// rebuild x3
```

Comment:
@matildepark it's changed, even if it doesn't make it to the built JS, it'll end up in the source map, which means the glob hash will be new.